<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US"><head><title>Jeff A. Bilmes Homepage</title><meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"></meta><meta name="Keywords" content="Jeff Bilmes, Bilmes"></meta><meta name="Description" content="Jeff Bilmes's Home Page."></meta><link rel="stylesheet" href="../bilmes_style.css" type="text/css"></link></head><body><div class="container"><div class="navigation"><ul class="navbar"><li><a href="index.html">Home</a></li><li><a href="about.html">About</a></li><li><a href="prospective.html">Prospective Students</a></li><li><a href="teaching.html">Teaching</a></li><li><a href="research.html">Research</a></li><li><a href="people.html">People</a></li><li><a href="sort_date.html">Publications</a></li><li><a href="software.html">Software 
                              &amp; Data</a></li><li><a href="talks.html">Talks</a></li></ul></div><div class="content"><div id="toptitle"><h1><a href="index.html">Jeff A. Bilmes</a>'s Publications</h1></div><p xmlns=""><big>&#8226;
  <a href="sort_date.html">Sorted by Date</a> &#8226;
  <a href="class_type.html">Classified by Publication Type</a> &#8226;
  <a href="class_rescat.html">Classified by Research Category</a> &#8226;
  <a href="sort_author.html">Sorted by First Author Last Name</a> &#8226;
  <a href="class_author.html">Classified by Author Last Name</a> &#8226;
  </big></p>
<h2><b> Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture</b></h2><p class="citation"> Jacob Schreiber,  Maxwell Libbrecht, <a href="http://melodi.ee.washington.edu/~bilmes"> Jeffrey Bilmes</a>, and  William Noble. <b> Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture</b>. <i> bioRxiv</i>,  Cold Spring Harbor Laboratory, 2017.</p><h3>Download</h3><p><a xmlns="" href="http://dx.doi.org/10.1101/103614">[PDF]</a>&nbsp;<a xmlns="" href="http://dx.doi.org/10.1101/103614">[gzipped postscript]</a>&nbsp;<a xmlns="" href="http://dx.doi.org/10.1101/103614">[postscript]</a>&nbsp;<a xmlns="" href="http://dx.doi.org/10.1101/103614">[HTML]</a>&nbsp;</p><h3>Abstract</h3><p class="abstract"> Recently, Hi-C has been used to probe the 3D chromatin architecture of multiple organisms and cell types. The resulting collections of pairwise contacts across the genome have connected chromatin architecture to many cellular phenomena, including replication timing and gene regulation. However, high resolution (10 kb or finer) contact maps remain scarce due to the expense and time required for collection. A computational method for predicting pairwise contacts without the need to run a Hi-C experiment would be invaluable in understanding the role that 3D chromatin architecture plays in genome biology. We describe Rambutan, a deep convolutional neural network that predicts Hi-C contacts at 1 kb resolution using nucleotide sequence and DNaseI assay signal as inputs. Specifically, Rambutan identifies locus pairs that engage in high confidence contacts according to Fit-Hi-C, a previously described method for assigning statistical confidence estimates to Hi-C contacts. We first demonstrate Rambutan\textquoterights performance across chromosomes at 1 kb resolution in the GM12878 cell line. Subsequently, we measure Rambutan\textquoterights performance across six cell types. In this setting, the model achieves an area under the receiver operating characteristic curve between 0.7662 and 0.8246 and an area under the precision-recall curve between 0.3737 and 0.9008. We further demonstrate that the predicted contacts exhibit expected trends relative to histone modification ChIP-seq data, replication timing measurements, and annotations of functional elements such as promoters and enhancers. Finally, we predict Hi-C contacts for 53 human cell types and show that the predictions cluster by cellular function.</p><a href="Schreiber103614.bib"><h3>BibTeX</h3></a><pre>@article {Schreiber103614,
	author = {Schreiber, Jacob and Libbrecht, Maxwell and Bilmes, Jeffrey and Noble, William},
	title = {Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture},
	year = {2017},
	doi = {10.1101/103614},
	publisher = {Cold Spring Harbor Laboratory},
	abstract = {Recently, Hi-C has been used to probe the 3D chromatin architecture of multiple organisms and cell types. The resulting collections of pairwise contacts across the genome have connected chromatin architecture to many cellular phenomena, including replication timing and gene regulation. However, high resolution (10 kb or finer) contact maps remain scarce due to the expense and time required for collection. A computational method for predicting pairwise contacts without the need to run a Hi-C experiment would be invaluable in understanding the role that 3D chromatin architecture plays in genome biology. We describe Rambutan, a deep convolutional neural network that predicts Hi-C contacts at 1 kb resolution using nucleotide sequence and DNaseI assay signal as inputs. Specifically, Rambutan identifies locus pairs that engage in high confidence contacts according to Fit-Hi-C, a previously described method for assigning statistical confidence estimates to Hi-C contacts. We first demonstrate Rambutan{\textquoteright}s performance across chromosomes at 1 kb resolution in the GM12878 cell line. Subsequently, we measure Rambutan{\textquoteright}s performance across six cell types. In this setting, the model achieves an area under the receiver operating characteristic curve between 0.7662 and 0.8246 and an area under the precision-recall curve between 0.3737 and 0.9008. We further demonstrate that the predicted contacts exhibit expected trends relative to histone modification ChIP-seq data, replication timing measurements, and annotations of functional elements such as promoters and enhancers. Finally, we predict Hi-C contacts for 53 human cell types and show that the predictions cluster by cellular function.},
	URL = {https://www.biorxiv.org/content/early/2017/01/30/103614},
	eprint = {https://www.biorxiv.org/content/early/2017/01/30/103614.full.pdf},
	journal = {bioRxiv},
}
</pre><h3>Share</h3><p class="share"><div class="addthis_toolbox addthis_default_style addthis_32x32_style"><a class="addthis_button_preferred_1"></a><a class="addthis_button_preferred_2"></a><a class="addthis_button_preferred_3"></a><a class="addthis_button_preferred_4"></a><a class="addthis_button_compact"></a></div><script type="text/javascript" src="http://s7.addthis.com/js/250/addthis_widget.js#pubid=xa-4e39e6cc4ff7d9d9"></script></p><hr width="100%" size="2"></hr><p><small>
 Generated by
 <a xmlns="" href="https://sourceforge.net/projects/bib2html/">bib2html.pl</a>
 (written by <a xmlns="" href="http://sourceforge.net/users/patstg/">Patrick Riley</a>
  ) on
  Sun Oct 01, 2023 14:42:07</small></p></div><div class="footer"><a class="right_pad" href="index.html">
Go home</a><a class="right_pad" href="http://mailhide.recaptcha.net/d?k=01l3ho3qEvdj--sJ6OJ8LS1A==&amp;c=7CXXfzP4KrBHZwLeD1xbEluzWyfYR5lzjGozsG53sBw=" onclick="window.open('http://mailhide.recaptcha.net/d?k=01l3ho3qEvdj--sJ6OJ8LS1A==&amp;c=7CXXfzP4KrBHZwLeD1xbEluzWyfYR5lzjGozsG53sBw=', '', 'toolbar=0,scrollbars=0,location=0,statusbar=0,menubar=0,resizable=0,width=500,height=300'); return false;" title="Reveal this e-mail address">Email me</a> Last updated Wed Jun 07 01:03 PDT 2023
 

</div></div></body></html>
